libs/testing/cypress/node/src/lib/plugins/auth.ts

```diff
@@ -1,20 +1,38 @@
 import { auth } from '../testing-cypress';
+import { USER_FIXTURES_PASSWORD } from '@blockframes/devops';
+import { serverId } from '@blockframes/utils/constants';
+import { User } from 'firebase/auth';
 
-export async function validateAuthUser(email: string) {
-  const user = await auth.getUserByEmail(email);
-  return await auth.updateUser(user.uid, { emailVerified: true });
+export async function createUser(data: { uid: string; email: string }) {
+  return await auth.createUser({ ...data, password: USER_FIXTURES_PASSWORD });
 }
 
-export async function getAuthUserByEmail(email: string) {
+export async function getUser(emailOrUid: string) {
   try {
-    return await auth.getUserByEmail(email);
+    return emailOrUid.includes('@') ? await auth.getUserByEmail(emailOrUid) : await auth.getUser(emailOrUid);
   } catch (err) {
     if (err.code === 'auth/user-not-found') return null;
-    throw err;
+    throw err.code;
   }
 }
 
-export async function deleteAuthUser(userId: string) {
-  await auth.deleteUser(userId);
-  return `User ${userId} deleted`;
+export async function deleteUser(uid: string) {
+  return await auth.deleteUsers([uid]);
+}
+
+export async function updateUser(data: { uid: string; update: Partial<User> }) {
+  const { uid, update } = data;
+  return await auth.updateUser(uid, update);
```

Comment:
you don't need `await` if you return
```suggestion
  return auth.updateUser(uid, update);
```

Reply:
As discussed, this a task, the `await` is needed